src/AsyncGenerator.yml

```diff
@@ -180,6 +178,8 @@
       hasAttributeName: IgnoreAttribute
     - conversion: NewType
       hasAttributeName: TestFixtureAttribute
+    - conversion: NewType
+      anyBaseTypeRule: HasTestFixtureAttribute
```

Comment:
Required for generating child classes of a base fixture if they are not themselves marked with the `[Fixture]` attribute.